webpack/webpack.config.js

```diff
@@ -3,7 +3,10 @@ module.exports = {
   entry: './app/main.js',
   output: {
     filename: 'nitf.js',
-    path: '../src/collective/nitf/static'
+    path: '../src/collective/nitf/static',
+    libraryTarget: 'umd',
```

Comment:
Use webpack to create a library instead of bundle application https://webpack.js.org/guides/author-libraries/